src/api/app/models/project.rb

```diff
@@ -350,7 +350,7 @@ def self.get_maintenance_project(at = nil)
     at ||= AttribType.find_by_namespace_and_name!('OBS', 'MaintenanceProject')
     maintenanceProject = Project.find_by_attribute_type(at).first
     unless maintenanceProject and check_access?(maintenanceProject)
-      raise UnknownProject.new 'There is no project flagged as maintenance project on server and no target in request defined.'
+      raise BsRequestAction::UnknownProject.new 'There is no project flagged as maintenance project on server and no target in request defined.'
```

Comment:
I'm with you not having basic models and not using exceptions there... but it was like it is no real changes from my side. Just added the namespace ```BsRequestAction``` because ```UnknownProject``` is only defined there. Just fixing because it was broken, now we have at least an exception class and it "works". 

What solution do you propose?  :smiley_cat: 
Creating the `UnknownProject` exception class inside Project model and use it there? 
Get rid of that check and just return nil and have this in mind when is called? (this method is called at source_controller, bs_request:_action_maintenance_incident model and channel_binary model)